docs/features/api.md

```diff
@@ -0,0 +1,53 @@
+---
+title: API instructions
+---
+
+# intro
+
+> skip the intro if you are familiar with the datapackage concept.
+
+One of ideas of the datahub.io project is to store the data together with its description (the metadata). Data+metadata are called the *datapackage* or *dataset*. Detailed description of the *datapackage* format is here: http://datahub.io/docs/data-packages
```

Comment:
I will leave the Intro here, because I remember my confusing and my questions when I a first met all this concepts, like "descriptor" (descriptor of what?), datapackage (is this a zip-archive?), dataset (what the difference from datapackage) and many more stupid questions.

This few sentences 
- could be skipped
- answers the question "why is that API?", which makes the article more solid
- doesn't take too much space (6 lines of well structured text)

Reply:
@AcckiyGerman I'm afraid it can confuse people as this might not be "one of ideas of datahub". Also, this kind of introduction should be in the "introduction" document or somewhere, but not in the middle of advanced docs.